### PR TITLE
Docs Readme update

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**

### CODE REVIEW

**Comment by orenyodfat** [comment]
Please remove "ArcHives - A public curated database of elements, organizations and a shared place for community collaboration." (or explain).

**Comment by orenyodfat** [comment]
> Vanille (and more...) - Collaborative DApps, built on top of DAOstack using Arc.js

might consider to change to something like  : 

> Collaborative DApps, built on top of DAOstack using Arc.js .e.g Vanille ,Alchemy...

File: docs/README.md [hunk 1]
```diff
@@ -6,17 +6,13 @@ Check out the [Generated docs](generated_docs) for detailed documentation on eve
...
-
-The DAOStack:
+The DAO stack:
 
 1. [Ethereum](https://www.ethereum.org/) - *Base blockchain.*
-2. ArcHives - *A public curated database of [element](#elements)s, organizations and a shared place for community collaboration*.
-3. **Arc - This project**
 4. [Arc.js](https://github.com/daostack/arc.js) - *JavaScript library that talks to Arc, built on top of web3.js.*
-5. [Vanille](https://github.com/daostack/vanille) (and more...) - *Collaborative DApps, built on top of DAOstack using Arc.js*
+5. [Vanille](https://github.com/daostack/vanille) & [Alchemy](https://github.com/daostack/alchemy) - *Collaborative DApps, built on top of DAOstack using Arc.js*
```
Comment (orenyodfat) [inline docs/README.md]: Vanille and Alchemy are 2 from many potential apps on top arc.js and arc.
I will change that to  : 
 -Collaborative DApps, built on top of DAOstack using Arc.js. 
For example Vanille and Alchemy

Comment (dev-matan-tsuberi) [reply docs/README.md]: ok

**Comment by orenyodfat** [comment]
> !!! note "yarn docs preview does not serve the final website!" ..

Should it be-  Note : yarn docs preview does not serve the final website. 

I will also add a link to the website and call that docs website or something more specific

**Comment by dev-matan-tsuberi** [comment]
@orenyodfat 
No. This synthax is a markdown extenstion called [admonition](https://squidfunk.github.io/mkdocs-material/extensions/admonition/#usage), take a look at the result in the [website](https://daostack.github.io/arc/#contributing-to-arc-docs).

I realise it comprimises the github readability a little bit, but it more than compensates on the website, which should be the main place for docs i think.

**Comment by leviadam** [comment]
Hi @dev-matan-tsuberi, I think this file should be more elaborative. And describe the architecture.
An image will also help.

**Comment by dev-matan-tsuberi** [comment]
@leviadam 
Updated: https://github.com/daostack/arc/pull/309/commits/f8418f12f872cc7bea118b6ab52d0afadb5fc69c

Is there anything more you think we should add? + Is the information reflecting Arc correctly?

File: docs/README.md (hunk 1 above)
Comment (dev-matan-tsuberi) [reply docs/README.md]: changed in https://github.com/daostack/arc/pull/309/commits/f8418f12f872cc7bea118b6ab52d0afadb5fc69c

**Comment by dkent600** [comment]
Just to make sure you're aware:  The arc.js package is at @daostack/arc.js now

**Comment by dev-matan-tsuberi** [comment]
@dkent600 How is this related?

File: docs/README.md [hunk 2]
```diff
@@ -1,55 +1,59 @@
...
+3. **Security** - *Arc* embodies the best security guidelines and is very well tested.
+2. **Interoperability & Compositionality** - *Arc* can integrate with any other ethereum based framework with ease. Organizations can interact with each other and form hierarchies and other complex structures.
+1. **Modularity** - *Arc* is built in a modular way allowing anyone to extend and expand the ecosystem to support new use-cases.
+2. **General purpose** - *Arc* provides a single unified simple model for building any DAO imaginable.
 
-For more: [Read the official DAOstack Whitepaper](https://github.com/daostack/docs/blob/master/DAOstack%20White%20Paper%20V1.0.pdf).
+For more information [read the official DAOstack Whitepaper](https://github.com/daostack/docs/blob/master/DAOstack%20White%20Paper%20V1.0.pdf).
 
-Arc is the lower layer of the DAOStack. It consists of several smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+Check out the [Generated docs](generated_docs) for detailed documentation on every contract and an explanation of the source tree.
```
Comment (orenyodfat) [inline docs/README.md]: Add a link and explanation to the manual documentation (contracts) also ..

File: docs/README.md [hunk 3]
```diff
@@ -1,55 +1,59 @@
...
+For each DAO, the above components are organized in a single smart contract called a *[Controller](generated_docs/controller/Controller)*, which acts as the owner of the other components.
 
 ## Elements
 
 The main configurable and extendable building blocks of DAOs are:
 *Schemes* and *Global Constraints*, which are collectively called *elements*.
 
-* **[Schemes](generated_docs/universalSchemes/UniversalSchemeInterface.md)** *define the "do's" of the DAO, defining rules under which actions are taken, based on the participants input.* Examples of Schemes are:
-    * Rewarding contributions if other participants recognize them as worthy.
+* **[Schemes](generated_docs/universalSchemes/UniversalSchemeInterface.md)** *define the "dos" of the DAO, the rules under which actions are taken, based on the participants' input.* Examples of Schemes are:
```
Comment (orenyodfat) [inline docs/README.md]: should this link be replaced with a link to contracts/universalSchemes/README.md ?
which should give details explanation of u schemes?

File: docs/README.md [hunk 4]
```diff
@@ -1,55 +1,59 @@
...
+    * Rewarding contributions when other participants recognize them as worthy.
     * Voting on proposals and automatically executing the chosen proposal.
-* **[Global constraints](generated_docs/globalConstraints/GlobalConstraintInterface.md)** *define the "don't" of the DAO, they define conditions that must always hold and can prevent an action from being taken if it violates these conditions.* Examples of Global Constraints are:
+* **[Global constraints](generated_docs/globalConstraints/GlobalConstraintInterface.md)** *define the "don'ts" of the DAO, the conditions that must always hold and can prevent an action from being taken if it violates these conditions.* Examples of Global Constraints are:
```
Comment (orenyodfat) [inline docs/README.md]: should this link be replaced with a link to contracts/globalConstraints/README.md ?
which should give details explanation of globalConstraints?
Same for controller and voting machines.

File: docs/README.md (hunk 2 above)
Comment (dev-matan-tsuberi) [reply docs/README.md]: The generated docs also include the manual docs as headers. 
I think it's better to ditch the headers thing, and just link from the generated docs from the manual docs. There will be no duplication then. 
What do you think?

Comment (dev-matan-tsuberi) [reply docs/README.md]: updated at: https://github.com/daostack/arc/pull/309/commits/ec72cd04e458beaad55e670c756d3a254a6be84c

If this is not wanted, we can undo this commit.

File: docs/README.md (hunk 4 above)
Comment (dev-matan-tsuberi) [reply docs/README.md]: updated at: https://github.com/daostack/arc/pull/309/commits/ec72cd04e458beaad55e670c756d3a254a6be84c

File: docs/README.md (hunk 3 above)
Comment (dev-matan-tsuberi) [reply docs/README.md]: updated at: https://github.com/daostack/arc/pull/309/commits/ec72cd04e458beaad55e670c756d3a254a6be84c

**Comment by dkent600** [comment]
@dev-matan-tsuberi 

> Just to make sure you're aware: The arc.js package is at @daostack/arc.js now
> How is this related?

Sorry Matan, I don't recall.  I must've seen something that I either misinterpreted or has since been corrected.

**Comment by dev-matan-tsuberi** [comment]
@leviadam 
> what is this warning in the doc?

It's an example of how to check for broken links by using `npm run docs preview`.

**Comment by leviadam** [comment]
Two things. I think the contribution for docs part is too long, but we can live with it.
The warning itself I would make less sticking out. Just write to look for error for broken links. It grabs too much attention like this. We have to make sure the attention of the reader in the readme on the main issues.

**Comment by dev-matan-tsuberi** [comment]
@leviadam updated: https://github.com/daostack/arc/pull/309/commits/c29204906a57d9761fd9650367d4fdc0f85ac36a

**Comment by leviadam** [comment]
I'm approving, but please update `yarn` to `npm` before merging.

**Pull request merged**